### ISSUE TITLE
COPY / DUPLICATE Function

### ISSUE THREAD

**Issue by Francky63** [WhiteMagic/JoystickGremlin#102]
When you have a lot of rules to apply related the same function on different button could be useful to have a COPY/DUPLICATE function.
I mean, you have to build up a container whit more remap with time and condition or macro for one button as a target other vJoy buttons.
If you have to make the same rules for other buttons with other targets but same rules you can copy it on the new source button,Axis, keys,, and the change only the destinations buttons, axis.
Now you have to rewrite boring rules or copy and paste from XML profile file

**Comment by WhiteMagic** [comment]
Yeah I had that on my list of things I want to eventually add for a while. Thinking about it quickly tells me this should be fairly easy to add, though there probably is something that will make it slightly harder then I'm anticipating now.

**Comment by WhiteMagic** [comment]
R14 supports this now via the Reference action.
